src/node/utils/Settings.js

```diff
@@ -430,6 +430,11 @@ exports.importMaxFileSize = 50 * 1024 * 1024;
  */
 exports.enableAdminUITests = false;
 
+/*
+ * Enable case-insensitive pad names.
+ */
+exports.caseInsensitivePadNames = false;
+
```

Comment:
Superfluous newline:

```suggestion
```

Reply:
applied in 68fa4f19c45f23c01c8f6f0c54793f78cd1ce930